src/SFA.DAS.SharedOuterApi/Interfaces/IAssessorsApiClient.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.SharedOuterApi.Interfaces
+{
+    public interface IAssessorsApiClient<T> : IInternalApiClient<T>
```

Comment:
There is an `IGetApiClient<T>` been added which might be better for this?

Reply:
yeah will do.